pokemongo_bot/cell_workers/incubate_eggs.py

```diff
@@ -106,15 +106,13 @@ def _check_inventory(self, lookup_ids=[]):
                         "used": False
                     })
                 elif 'is_egg' not in pokemon and pokemon['id'] in lookup_ids:
-                    matched_pokemon.append({
-                        "pokemon_id": pokemon.get('pokemon_id', -1),
-                        "cp": pokemon.get('cp', 0),
+                    matched_pokemon.append(pokemon.update({
```

Comment:
If it isn't *needed* by this PR, can you undo these changes? It makes it much scarier to review and accept these PRs since it is hard to tell what the impact across the whole project is going to be. Keeping changes small and focused on a single worker makes our lives way easier. If we need these changes down the road, I'd feel more comfortable making them when it actually impacts something.